src/views/welcome/ChangelogScreen.tsx

```diff
@@ -143,7 +143,7 @@ const ChangelogScreen: Screen<"ChangelogScreen"> = ({ route, navigation }) => {
               Impossible de trouver les notes de mise à jour
             </NativeText>
             <NativeText variant="subtitle">
-              Vous êtes peut-être hors-ligne ou alors quelque chose ne s'est pas passé comme prévu...
+              Tu es peut-être hors-ligne ou alors quelque chose ne s'est pas passé comme prévu...
```

Comment:
```suggestion
              Tu es peut-être hors-ligne ou alors une erreur est survenue...
```
J'ai pris 1 heure à chercher si passé avait un e en plus, alors j'ai changé la phrase 😂